packages/typespec-test/test/openai_non_branded/generated/typespec-ts/README.md

```diff
@@ -0,0 +1,21 @@
+# OpenAI REST client library for JavaScript
+
+The OpenAI REST API. Please see https://platform.openai.com/docs/api-reference for more details.
+
+Key links:
+
+- [Package (NPM)](https://www.npmjs.com/package/@msinternal/openai-non-branded)
+
+## Getting started
+
+### Currently supported environments
+
+- LTS versions of Node.js
+
+### Install the `@msinternal/openai-non-branded` package
+
+Install the OpenAI REST client REST client library for JavaScript with `npm`:
+
+```bash
+npm install @msinternal/openai-non-branded
```

Comment:
Metadata files improvement should be covered in issue https://github.com/Azure/autorest.typescript/issues/2100